tests/NHSD.BuyingCatalogue.Solutions.Persistence.DatabaseTests/SolutionDetailRepositoryTests.cs

```diff
@@ -371,15 +347,11 @@ await CatalogueItemEntityBuilder
 
             await SolutionEntityBuilder.Create()
                 .WithId(Solution1Id)
-                .Build()
-                .InsertAsync();
-
-            await SolutionDetailEntityBuilder.Create()
                 .WithHosting(expectedHostingString)
                 .Build()
                 .InsertAndSetCurrentForSolutionAsync();
```

Comment:
As above, we should call `InsertAsync` instead of `InsertAndSetCurrentForSolutionAsync`.

Reply:
Replaced `InsertAndSetCurrentForSolutionAsync` with `InsertAsync`